<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Level</class>
 <widget class="QWidget" name="Level">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1002</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1002</width>
    <height>900</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1002</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="keyboard">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>851</width>
     <height>281</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>851</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-image: url(&quot;/home/olga/Kursovaya/Resourses/Keyboard.png&quot;) no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Keyboard.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
    border: none;
    outline: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resourses.qrc">
     <normaloff>:/Resourses/icons8-pause-button-100.png</normaloff>:/Resourses/icons8-pause-button-100.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>140</width>
     <height>140</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+V</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-image: url(&quot;/home/olga/Kursovaya/Resourses/timer.png&quot;) no-repeat right bottom;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>timer.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="time_spend">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EB Garamond</family>
     <pointsize>26</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>0:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="progress">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EB Garamond</family>
     <pointsize>26</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>0/245</string>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>821</width>
     <height>321</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>30</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>500</y>
     <width>821</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lang">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mistake1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mistake2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mistake3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>851</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <zorder>keyboard</zorder>
  <zorder>timer</zorder>
  <zorder>time_spend</zorder>
  <zorder>progress</zorder>
  <zorder>pushButton</zorder>
  <zorder>text</zorder>
  <zorder>textEdit</zorder>
  <zorder>lang</zorder>
  <zorder>mistake1</zorder>
  <zorder>mistake2</zorder>
  <zorder>mistake3</zorder>
  <zorder>verticalLayoutWidget</zorder>
 </widget>
 <resources>
  <include location="Resourses.qrc"/>
 </resources>
 <connections/>
</ui>
